packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -158,7 +159,16 @@ export class EpochTracker {
             this.logger.sendErrorEvent({ eventName: "EpochVersionMismatch" }, error);
             assert(!!this._hashedDocumentId, "DocId should be set to clear the cached entries!!");
             // If the epoch mismatches, then clear all entries for such document from cache.
-            await this.persistedCache.removeAllEntriesForDocId(this._hashedDocumentId);
+            const entry: ICacheEntry = {
+                file: {
+                    docId: this._hashedDocumentId,
+                    // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+                    resolvedUrl: {} as IOdspResolvedUrl,
```

Comment:
That does not look right. Can we instead ass whole file entry to this layer, not just hashedDocumentId?
This would be ideal - keep API consistent and allowing hosts to store / index data on whatever field it likes.

Reply:
Suppied file entry now instead of just docId.